src/chocolatey/infrastructure.app/commands/ChocolateyConfigCommand.cs

```diff
@@ -71,7 +71,7 @@ public virtual void ParseAdditionalArguments(IList<string> unparsedArguments, Ch
             if ((configuration.ConfigCommand.Command == ConfigCommandType.List
                  || !string.IsNullOrWhiteSpace(configuration.ConfigCommand.Name)
                 )
-                && unparsedArguments.Count > 1) throw new ApplicationException("A single features command must be listed. Please see the help menu for those commands");
+                && unparsedArguments.Count > 1) throw new ApplicationException("A single feature command must be listed. Please see the help menu for those commands.");

```

Comment:
Should this be?

```suggestion
                && unparsedArguments.Count > 1) throw new ApplicationException("A single config command must be listed. Please see the help menu for those commands.");
```

Since this is in the `ChocolateyConfigCommand`.

Reply:
Good catch. Fixed that now.